SampleApp/Project.swift

```diff
@@ -33,7 +33,25 @@ let project = Project(
         // does not currently support creating targets for tests in SwiftPM dependencies. See
         // https://github.com/tuist/tuist/issues/5912
 
-        .unitTest(for: "BlueprintUI"),
+        .app(
+            name: "TestHost",
+            sources: ["TestHost/**"]
+        ),
+
+        .unitTest(
+            for: "BlueprintUI"
+        ),
+
+        .target(
```

Comment:
This adds a UITest that runs against a test host. Notably this is used to verify UIKIt, not Blueprint.

This is needed to ensure that UIKit elements have their accessibility values set properly which happens lazily when running in a vanilla XCTest.